src/components/Header/Header.js

```diff
@@ -21,14 +20,17 @@ const Header = () => (
       <PageSettings />
     </div>
     <div className={`${CN}__heading-search-cart-container content`}>
-      <Link to="/home">
+      <Link to="/home" className={`${CN}__title-wrapper`}>
         <h2 className={`${CN}__title`}>The shop</h2>
       </Link>
       <div className={`${CN}__buttons-container`}>
         <BurgerMenu menuItemsList={burgerMenuItemsList} />
-        <Input icon="search" customClass={`${CN}__search-input`}>Search</Input>
+        <Input icon="search" placeholder="SEARCH" className={`${CN}__search-input`}>Search</Input>
```

Comment:
children - not passed as property inside `<Input>`, and no where to be found which is incorrect